.github/workflows/ci.yaml

```diff
@@ -75,7 +75,11 @@ jobs:
         java-version: 8
     - run: java -version
     - run: .kokoro/install_dependencies.sh
-    - run: .kokoro/linkage-monitor.sh
```

Comment:
This is the background of this change. This linkage-monitor.sh is being removed by synthtool ([the change in flight](https://github.com/googleapis/google-api-java-client/pull/1717/files#diff-4f244f13abe613fb9c24ebb3dd11e7c9a2e32fb6442dd07a2cb01ad85cbe5339L1)). We should use Linkage Monitor GitHub Action ("GoogleCloudPlatform/cloud-opensource-java/linkage-monitor@v1-linkagemonitor")